core/templates/domain/topic/TopicObjectFactory.ts

```diff
@@ -228,6 +236,15 @@ export class Topic {
         'Topic url fragment should not be longer than ' +
         `${topicUrlFragmentCharLimit} characters.`);
     }
+    if (this._skillsForDiagnosticTest.length === 0) {
+      issues.push(
+        'Skill for the diagnostic test in the topic should not be empty.');
```

Comment:
The diagnostic test for the topic should test at least one skill.

Reply:
Updated